start-local.sh

```diff
@@ -5,4 +5,5 @@ set -o pipefail
 set -o nounset
 set -o xtrace
 
+cd /code
```

Comment:
It's not required. Scripts passed to `CMD` should be dead simple and their only responsibility should be to start some service (not copy code - it should be done in Dockerfile and/or by using docker volumes/mounts)